script/git-clone-others.sh

```diff
@@ -89,6 +89,7 @@ done
 
 
 for repository in $REPOSITORY_LIST ; do
+    gitUrlPrefix=${droolsjbpmGitUrlPrefix}
```

Comment:
@afalhambra why this was taken out if the second if statement?

Reply:
to make it available for all the different branches in the if conditions below, where this var can be used.
For instance:

https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/2c2fdf86e2c3cd1488c26b05ec89449fff3438e0/script/git-clone-others.sh#L128

and

https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/2c2fdf86e2c3cd1488c26b05ec89449fff3438e0/script/git-clone-others.sh#L120